pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -2006,7 +2006,7 @@ private void internalCreateSubscriptionForNonPartitionedTopic(AsyncResponse asyn
     }
 
     protected void internalResetCursorOnPosition(AsyncResponse asyncResponse, String subName, boolean authoritative,
-            MessageIdImpl messageId) {
+            MessageIdImpl messageId, boolean isExclusive) {
```

Comment:
exclude? I think exclude is more reasonable here.

Reply:
I think it might be more appropriate to change to isExclued.I am modifying it.